### PR TITLE
List collections for subjects.  Closes #1833

### LINKED ISSUE #1833: List collections containing the subject
May need a new API for /collections_subjects in Panoptes.

### PR DESCRIPTION
**Pull request by parrish**

### CODE REVIEW

File: app/collections/preview.cjsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          {if @state.owner
+            <Link className="user-profile-link" to="user-profile" params={name: @state.owner.login}>
+              <Avatar user={@state.owner} />{' '}{@state.owner.display_name}
+            </Link>}
+        </p>
+        <div className="subject-previews">
+          {if @state.subjects
+            <div>
+              {for subject in @state.subjects
+                <Thumbnail src={getSubjectLocation(subject).src} width={100} />}
```
Comment (aweiksnar) [inline app/collections/preview.cjsx]: Can you add in a key prop to these?

Comment (parrish) [inline app/collections/preview.cjsx]: Updated!

Comment (aweiksnar) [inline app/collections/preview.cjsx]: Thanks! Looks great

**Comment by aweiksnar** [comment]
:shipit:

**Pull request merged**